### ISSUE TITLE
List mediaItems without the readDevData authorization scope

### ISSUE THREAD

**Issue by axelguilmin** [deivitaka/GPhotos#2]
I love this library 👍 
I use it to list users pictures, in a purely read-only fashion, my code looks like this :
```
GPhotos.initialize(with: Config())
GPhotos.authorize(with: [.openId, .readOnly], completion: { _, _ in
    GPhotosApi.mediaItems.list(completion: { _ in })
})
```

This asks the user for 2 authorizations (`.readOnly` and `.readDevData`) :
<img width="315" alt="Screenshot 2020-07-05 at 16 17 53" src="https://user-images.githubusercontent.com/1522643/86534739-1654e880-bedb-11ea-90b4-10cb27e6b60e.png">
But I don't need the `.readDevData` authorization (at least I think so)

In `GPhotosResource.swift` :
```
internal struct scopes {
    static let read: ScopeSet = ([.readOnly, .readDevData], [.readAndAppend])
```
Both are required, so changing the line 16 to this would solve my issue :
```
static let read: ScopeSet = ([.readOnly], [.readDevData, .readAndAppend])
```
However I'm not sure this is the way to go, is it the wanted behavior to always ask for `.readDevData` alongside `.readOnly` ?

**Comment by deivitaka** [comment]
Valid point. I treated the `read` scope as read-only access to all photos, user-added or added from your app. But the latter should be optional considering your use case. Thanks for pointing out!

**Comment by deivitaka** [comment]
Fixed in ac1e0a6

**Comment by axelguilmin** [comment]
Awesome thank you very much!
